lib/game/embedded_game_widget.dart

```diff
@@ -15,14 +15,19 @@ class EmbeddedGameWidget extends LeafRenderObjectWidget {
   final Game game;
   final Vector2 size;
 
-  EmbeddedGameWidget(this.game, {this.size});
+  EmbeddedGameWidget(
+    this.game, {
+    this.size,
```

Comment:
Isn't this conflicting with a previous PR of yours?

Reply:
Yep, but this one has priority.

Reply:
that one*